MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -307,12 +307,22 @@ public void Present()
             PlatformPresent();
         }
 
-        /*
+#if WINDOWS && DIRECTX
         public void Present(Rectangle? sourceRectangle, Rectangle? destinationRectangle, IntPtr overrideWindowHandle)
         {
-            throw new NotImplementedException();
+            // TODO: How to handle sourceRectangle and destinationRectangle?
+            SwapChainRenderTarget scrt;
+            if (!_overrideWindows.TryGetValue(overrideWindowHandle, out scrt))
+            {
+                scrt = new SwapChainRenderTarget(this, overrideWindowHandle, PresentationParameters.BackBufferWidth, PresentationParameters.BackBufferHeight);

```

Comment:
Note from everything I can find DX11 does not support the DX9 style... present to a different front buffer... method.

IMO this should simply throw a not supported exception and suggest people use `SwapChainRenderTarget` directly.